.github/CHANGELOG.md

```diff
@@ -15,6 +15,9 @@
 
 ### Improvements
 
+* Add `sample(wires)` support in LightningQubit.
```

Comment:
@vincentmr  current me if I'm wrong here: `sample(wires)` is still supported as before in Python. You added a binding for `generate_samples` and improved the C++ implementation of `generate_samples(wires)`. The Python support will be added in a followup PR as discussed offline. 
```suggestion
* Add `generate_samples(wires)` support in LightningQubit.
```

Reply:
Correct.